pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
> sgtm. I wanted to distinguish between the list that we use for initializing the kube master and the openshift master. Is it ok to name the other list openshiftAdmissionPluginOverride ?

I'm not convinced that we want separate lists long-term.  We do that currently, but do we have a good reason for doing it?  Admission plugins already whitelist what they deal with.